src/App.jsx

```diff
@@ -14,33 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const sortAlphabetical = 'name';
+const sortLength = 'length';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const [isChanged, setIsChanged] = useState(false);
+
+
+  let sortedListGoods = getPrepareGoods(goodsFromServer, sortField, isReversed);
+
+  function getPrepareGoods(goods, sortField, isReversed) {
+    let SortedGoods = [...goods];
+    if (sortField) {
+      SortedGoods.sort((good1, good2) => {
+        switch (sortField) {
+          case sortAlphabetical:
+            return good1.localeCompare(good2);
+          case sortLength:
+            return good1.length - good2.length;
+          default:
+            return 0;
+        }
```

Comment:
```suggestion
        switch (sortField) {
          case sortAlphabetical:
            sortedGoods.sort((good1, good2) =>good1.localeCompare(good2));
            break;
          case sortLength:
            sortedGoods.sort((good1, good2) => good1.length - good2.length);
            break
          default:
            break;
        }
```
in defautl redundant render. make sort in each case, and by default break.